src/modules/mtev_amqp.c

```diff
@@ -398,7 +398,7 @@ init_conns(void) {
       goto bail;
     }
   }
-  free(mqs);
+  mtev_conf_release_sections(mqs, the_conf->number_of_conns);
```

Comment:
the goto on line 398 skips this `mtev_conf_release_sections()`. line 405 probably needs to be updated